tests/integration/helpers.py

```diff
@@ -433,6 +436,21 @@ def needs_trust(self) -> bool:
         """
         return any(app.get("trust", False) for app in self.applications.values())
 
+    def get_base_by_series(self) -> str:
+        """Get the base release for the series.
+
+        Returns:
+            str: base release for the series
+        """
+        if not self.series:
+            return "22.04"
+
+        return {
+            "focal": "20.04",
+            "jammy": "22.04",
+            "noble": "24.04",
+        }.get(self.series, "22.04")
+
```

Comment:
while i love this... checkout `juju.utils.get_series_version(...)` and see if it doesn't meet your needs

Reply:
I didn't know this was a thing. Updated!